config.go

```diff
@@ -32,6 +43,24 @@ type Config struct {
 	// RequestShutdown is an optional closure to request clean shutdown
 	// from the calling entity if the boltnd instance errors out.
 	RequestShutdown func()
+
+	// LNDRetires is the number of times we try to connect to lnd's grpc
+	// server (with backoff set by LNDWait) before exiting with an error.
+	// This value should be set to 0 if we want to immediately exit if
+	// we can't successfully connect.
+	LNDRetires uint8
+
+	// LNDWait is the amount of time to wait between retries to connect to
+	// lnd's grpc server.
+	LNDWait time.Duration
+}
+
+// DefaultConfig returns a default config.
+func DefaultConfig() *Config {
```

Comment:
Explain that this is not valid as is + add todo to add valid default lnd config (or just make it valid).